src/Nancy/Responses/DefaultJsonSerializer.cs

```diff
@@ -100,6 +100,7 @@ private static bool IsJsonType(string contentType)
             return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
                 contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||
                 contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
+                contentMimeType.Equals("application/problem+json", StringComparison.OrdinalIgnoreCase) ||
                 (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&

```

Comment:
We already have `contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase)`. Can't we just remove the `contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&` part? I don't know why that'd be a requirement for a JSON serializer...